mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.purgeOrder = async (req, res) => {
+  try {
+    const result = await Order.findOneAndRemove({ _id: req.params.orderId });
+    if (!result) {
+      return res.status(404).json({ message: "Order not located" });
```

Comment:
The response message should be consistent throughout the codebase. In the `exports.create` function, the error response message is `error: errorHandler(error)`, but in this code snippet, it's `message: "Order not located"`. It would be better to use the same format for error responses throughout the code. Consider using `error: errorHandler("Order not located")` for consistency.